sys-apps/utempter/utempter-0.5.5.6-r1.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="${P%.*}"
+MY_P="${MY_PN}-${PV##*.}"
+
+inherit flag-o-matic rpm toolchain-funcs
+
+DESCRIPTION="App that allows non-privileged apps to write utmp (login) info"
+HOMEPAGE="https://www.redhat.com/"
+SRC_URI="mirror://gentoo/${MY_P}.src.rpm"
+S="${WORKDIR}/${MY_PN}"
+
+LICENSE="|| ( MIT LGPL-2 )"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86"
+
+RDEPEND="
+	acct-group/utmp
+	!sys-libs/libutempter
+"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-0.5.5.6-no_utmpx.patch
+	"${FILESDIR}"/${PN}-0.5.5.6-fix-build-system.patch
+)
+
+src_prepare() {
+	default
+	tc-export CC
+	append-cflags -Wall
+}
+
+src_install() {
+	local myemakeargs=(
+		LIBDIR="/usr/$(get_libdir)"
+		RPM_BUILD_ROOT="${ED}"
+	)
+
+	emake "${myemakeargs[@]}" install
+
+	dobin utmp
+
+	fowners root:utmp /usr/sbin/utempter
+	fperms 2755 /usr/sbin/utempter
+}
+
+pkg_postinst() {
```

Comment:
I think we should drop pkg_postinst. OpenRC and systemd already set the proper permissions on these files on boot.

Reply:
Removed, done.